staging/src/k8s.io/api/apps/v1beta1/types.go

```diff
@@ -41,7 +41,7 @@ type ScaleStatus struct {
 	// actual number of observed instances of the scaled object.
```

Comment:
```suggestion
	// replicas is the number of observed instances of the scaled object.
```

If you find other instances like this, please fix. The first word should match the name of the field (in `lowerCamelCase`)

Reply:
Sure, I'll fix it in the next week 👍🏻

Reply:
Hmm I've updated most of the prefixes in that file, but this review suggestion seems to enlarge this PR several times 😢 I've left some comment like they were because they are the same throughout the whole codebase, e.g https://github.com/kubernetes/kubernetes/blob/d1877f514a3b743bf61bb7d63556b847fbf94c25/staging/src/k8s.io/api/apps/v1beta1/types.go#L67 is the same in 20+ places, but technically should be adjusted.

Should I fix also the v1beta2 file or just revert the d1877f5 and introduce only fixes to the links? I've 3 other PRs where I only fix the links:
1. https://github.com/kubernetes/kubernetes/pull/116115
2. https://github.com/kubernetes/kubernetes/pull/116113
3. https://github.com/kubernetes/kubernetes/pull/116109
and 2 of them got LGTM'd.

I'm also fine with fixing those comments but I'd like to know the scope of this :D